.storybook/stories/CorporateCampaign.stories.js

```diff
@@ -0,0 +1,58 @@
+import Vue from 'vue';
+import StoryRouter from 'storybook-vue-router';
+import { withKnobs, text, object } from '@storybook/addon-knobs';
+
+// import plugins
+import kivaPlugins from '@/plugins';
+Vue.use(kivaPlugins);
+
+import apolloStoryMixin from '../mixins/apollo-story-mixin';
+// import kvAuth0StoryMixin from '../mixins/kv-auth0-story-mixin';
+
+import CorporateCampaignLanding from '@/pages/LandingPages/CorporateCampaign/CCLandingPage';
+import CampaignHeader from '@/components/ContentGroups/CampaignHeader';
+import CampaignStatus from '@/components/ContentGroups/CampaignStatus';
+
+export default {
+	title: 'Pages/CorporateCampaignLanding',
+	component: CorporateCampaignLanding,
+	args: {
+		headerAreaContent: {
```

Comment:
Since headerAreaContent only applies to CampaignHeader, and the other props apply to StatusRow, I recommend creating two separate story files.